src/Spec.ts

```diff
@@ -1108,10 +1115,13 @@ ${await utils.readFile(path.join(__dirname, '../js/multipage.js'))}
     );
   }
 
-  private addStyle(head: HTMLHeadElement, href: string) {
+  private addStyle(head: HTMLHeadElement, href: string, print: boolean = false) {
```

Comment:
Maybe
```suggestion
  private addStyle(head: HTMLHeadElement, href: string, media?: string) {
```
for the sake of avoiding boolean attributes?

Reply:
The only allowed values are `all`/`print`/`screen` but okay fine.